app/components/Resource/Resource.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router';
 import PropTypes from 'prop-types';
-import { AddressInfo, TodaysHours, PhoneNumber, ResourceCategories, Website, StreetView } from './ResourceInfos';
+import { AddressInfo, TodaysHours, PhoneNumber, ResourceCategories, Website, Email, StreetView } from './ResourceInfos';
```

Comment:
Given our existing component soup problem (difficult to find anything) not a fan of big multifile components, like here in ResourceInfos personally, but we can address this down the line.

Reply:
Agreed. Also if we could create more reusable components, that would help, instead of having all components being organized by "page" or "React route."